flagsmith/models.py

```diff
@@ -10,27 +12,41 @@
 @dataclass
 class BaseFlag:
     enabled: bool
-    value: typing.Union[str, int, float, bool, type(None)]
+    value: typing.Union[str, int, float, bool, type[None]]
```

Comment:
`type[None]` can be just `None`? Actually I'm not even sure all Python versions we support implement `__getitem__` for this buit-in.

Reply:
Yes, `type[None]` is supported by python 3.9+. Using just `None`